### ISSUE TITLE
Storybook addon accesibility not initialising, error in console. On Storybook 6.1.21

### ISSUE THREAD

**Issue by GeorgeWL** [storybookjs/storybook#16976]
**Describe the bug**
I've installed https://storybook.js.org/addons/@storybook/addon-a11y using the recommended method in the docs, but it does not ever load the Accessibility Panel, instead showing `Initializing...` and giving an error message in the console.
The colour vision simulator button does work however.

**To Reproduce**
Install onto storybook version 6.1.21, installation is successful. Go onto a story after running the app, accessibility tab shows, but only with message "Initialising"

look into logs and get the following error from a11yRunner.js 

```js
Uncaught (in promise) TypeError: globalWindow.__STORYBOOK_STORY_STORE__.loadStory is not a function
    at _callee3$ (a11yRunner.js:161)
    at tryCatch (runtime.js:63)
    at Generator.invoke [as _invoke] (runtime.js:293)
    at Generator.next (runtime.js:118)
    at asyncGeneratorStep (a11yRunner.js:5)
    at _next (a11yRunner.js:7)
    at a11yRunner.js:7
    at new Promise (<anonymous>)
    at a11yRunner.js:7
    at getParams (a11yRunner.js:194)
```

**System**
 ``` 
System:
    OS: Windows 10 10.0.19043
    CPU: (12) x64 Intel(R) Core(TM) i7-10850H CPU @ 2.70GHz
  Binaries:
    Node: 14.16.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.5 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.12 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (96.0.1054.43)
  npmPackages:
    @storybook/addon-a11y: ^6.4.9 => 6.4.9
    @storybook/addon-actions: 6.1.21 => 6.1.21
    @storybook/addon-docs: 6.1.21 => 6.1.21
    @storybook/addon-essentials: 6.1.21 => 6.1.21
    @storybook/addon-links: 6.1.21 => 6.1.21
    @storybook/react: 6.1.21 => 6.1.21
```
**Additional context**
Unsure if anything here may affect it, but we have a slightly modified storybook webpack to fix the fact that we use MDX for some files
```js
const path = require('path');

const storiesFileTypes = '*.@(stories|story).@(js|jsx|mdx)';
//Add rules from the webpack config
const defaultConfig = require('../webpack.config');
const createCompiler = require('@storybook/addon-docs/mdx-compiler-plugin');

module.exports = {
    stories: [`../src/**/**/${storiesFileTypes}`],
    addons: [
        '@storybook/addon-essentials',
        '@storybook/addon-docs',
        '@storybook/addon-links',
        '@storybook/addon-a11y'
    ],
    webpackFinal: async (config, { configType }) => {
        // `configType` has a value of 'DEVELOPMENT' or 'PRODUCTION'
        // You can change the configuration based on that.
        // 'PRODUCTION' is used when building the static version of storybook.
        config.module.rules = [
            ...defaultConfig.module.rules,
            {
                test: /\.(stories|story)\.mdx$/,
                use: [
                    {
                        loader: 'babel-loader'
                    },
                    {
                        loader: '@mdx-js/loader',
                        options: {
                            compilers: [createCompiler({})]
                        }
                    }
                ]
            }
        ];
        config.resolve.alias = {
            features: path.resolve(__dirname, '..', 'src', 'features')
        };
        // Return the altered config
        return config;
    }
};
```

**Comment by shilman** [comment]
@GeorgeWL if you're using version 6.4 of addon-a11y you should be using 6.4 of all the other storybook packages as well. you can upgrade to the latest using:

```
npx sb@latest upgrade
```

**Comment by GeorgeWL** [comment]
> @GeorgeWL if you're using version 6.4 of addon-a11y you should be using 6.4 of all the other storybook packages as well. you can upgrade to the latest using:
> 
> ```
> npx sb@latest upgrade
> ```

So it's just that the version of the add-on should match the version of storybook currently installed?

**Comment by shilman** [comment]
@GeorgeWL correct

**Comment by GeorgeWL** [comment]
Whilst I'm here, the docs are wrong and tell you to put a story into the `main.js`
